webapp/src/services/applicationForm/applicationForm.service.js

```diff
@@ -202,4 +204,73 @@ function getQuestionList(eventId) {
             status: error.response && error.response.status
         };
     });
-}
\ No newline at end of file
+}
+
+function getForEventDetails(eventId) {
```

Comment:
Nit: Please call this `getDetailsForEvent` because you're getting "application form details" for an event, not an application form for an "event details"